pkgs/by-name/tu/turbo-unwrapped/package.nix

```diff
@@ -0,0 +1,100 @@
+{
+  stdenv,
+  lib,
+  fetchFromGitHub,
+  protobuf,
+  rustPlatform,
+  autoPatchelfHook,
+  capnproto,
+  darwin,
+  extra-cmake-modules,
+  fontconfig,
+  lld,
+  # see comment below
+  # nix-update-script,
+  openssl,
+  pkg-config,
+  rust-jemalloc-sys,
+  zlib,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "turbo-unwrapped";
+  version = "2.0.9";
+
+  src = fetchFromGitHub {
+    owner = "vercel";
+    repo = "turbo";
+    rev = "v${version}";
+    hash = "sha256-hthuKGbHKTcv/wDIw9Ltcwl6jXWLhsCyqC76+vQyC5U=";
+  };
+
+  patches = [
+    # upstream uses nightly where lazy_cell is stable
+    ./enable-lazy_cell.patch
+  ];
+
+  cargoLock = {
+    lockFile = ./Cargo.lock;
+    outputHashes = {
+      "pathfinder_simd-0.5.3" = "sha256-94/qS5d0UKYXAdx+Lswj6clOTuuK2yxqWuhpYZ8x1nI=";
+    };
+  };
+
+  nativeBuildInputs =
+    [
+      autoPatchelfHook
+      capnproto
+      extra-cmake-modules
+      pkg-config
+      protobuf
+    ]
+    # https://github.com/vercel/turbo/blob/ea740706e0592b3906ab34c7cfa1768daafc2a84/CONTRIBUTING.md#linux-dependencies
+    ++ lib.optional stdenv.isLinux lld;
+
+  buildInputs =
+    [
+      fontconfig
+      openssl
+      rust-jemalloc-sys
+      zlib
+    ]
+    ++ lib.optionals stdenv.isDarwin (
+      with darwin.apple_sdk_11_0.frameworks;
+      [
+        CoreFoundation
+        CoreServices
+        IOKit
+      ]
+    );
+
+  cargoBuildFlags = [
+    "--package"
+    "turbo"
+  ];
+
+  # Browser tests time out with chromium and google-chrome
+  doCheck = false;
+
+  env = {
+    # nightly features are used
+    RUSTC_BOOTSTRAP = 1;
+  };
+
+  passthru = {
+    # TODO: The repo's releases.atom doesn't seem to have any stable releases? This breaks updating for now :/
+    # updateScript = nix-update-script { };
```

Comment:
is this needed?

Reply:
Not really, but it'd be nice in the future -- as it allows for the use of things like `./maintainers/scripts/update.nix`